packages/block-library/src/cover/style.scss

```diff
@@ -37,15 +37,19 @@
 	h2,
 	.wp-block-cover-image-text,
 	.wp-block-cover-text {
-		color: $white;
 		font-size: 2em;
 		line-height: 1.25;
 		z-index: 1;
 		margin-bottom: 0;
 		max-width: $content-width;
 		padding: $block-padding;
 		text-align: center;
+	}
 
+	h2:not(.has-text-color),
```

Comment:
Related to this #17704, why do we need to style h2? I guess this is not needed anymore right? (all the h2 references in this file). For example,  you'll notice that it's impossible to "left align" and h2 inside a Cover.